setup.py

```diff
@@ -165,6 +165,13 @@ def get_data_files(name, version, fullname):  # pylint: disable=R0912
         else:
             # sles 12+ and openSUSE 13.2+ use systemd
             set_systemd_files(data_files, dest=systemd_dir_path)
+    elif name == 'sles': # sles 15+ distro named as sles
```

Comment:
have you confirmed this comment? - 

># sles 15+ distro named as sles

I mean what does suse 13,14,etc do? Do we even have those distros? It would be wise to check all this out before committing it to code.

Reply:
we don't have 13.x, 14.x versions and also, [azure ](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/endorsed-distros)not endorsed those distros.